packages/ui/src/bounty/components/BountyListItem/components/DetailBox.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TextSmall } from '@/common/components/typography'
+import { Colors } from '@/common/constants'
+
+interface Props {
+  title: string
+  children: React.ReactNode | string | number
```

Comment:
I believe ReactNode already covers "string" and "number".``
```suggestion
  children: React.ReactNode
```